### ISSUE TITLE
download blocked videos on 1080p

### ISSUE THREAD

**Issue by bachbell** [rg3/youtube-dl#17404]
I have some blocked 1080p videos worldwide, i CAN see them on browser, i can download a 720p with plugins but cannot download full hd.

i just installed youtube-dl on a linux centos vps

i use this command and says this video is unavalaible, i really wish downad them on 1080p

youtube-dl -umygmauil@gmail.com -pmypass https://www.youtube.com/watch?v=g8FndVSahG8

thank you

**Comment by bachbell** [comment]
yes, i can watch the on vps firefox and able to download a 720p version but not 1080p. obvious logged in account

**Comment by Hrxn** [comment]
`ffmpeg, ffprobe` on VPS?

**Comment by bachbell** [comment]
hi, can you give me full example, im not a dev :/

**Comment by Hrxn** [comment]
You need both `ffmpeg` and `ffprobe` on your VPS, if you can run `youtube-dl` there it should work as well..

**Comment by bachbell** [comment]
i have ffmeg installed from repositories, is ffprobe a standalone?  i tried the command yum install ffprobe didnt work
